src/pages/docs/infrastructure/workers/index.mdx

```diff
@@ -38,13 +38,17 @@ The following steps always run inside the Octopus Server process (and do not run
 
 A worker receives instruction from the Octopus Server to execute a step, it executes the step using Calamari and returns the logs and any collected artifacts to the Octopus Server.
 
+:::div{.hint}
+Workers are assigned at the start of a deployment or runbook, not at the time the individual step executes.
+:::
+
 There are two kinds of workers you can use in Octopus:
 
-1. [The built-in worker](#built-in-worker) (default)
+1. [The built-in worker](#built-in-worker) (default for self-hosted Octopus)
 1. [External workers](#external-workers)
 
```

Comment:
Add 3rd item for "Dynamic Workers" (only available in Octopus Cloud). Also, table?